test/README.md

```diff
@@ -12,7 +12,7 @@ To run only the integration tests, run from the test directory:
 
 Name                            | Default                   | Description
 :--                             | :---                      | :----------
-LXD\_BACKEND                    | dir                       | What backend to test against (btrfs, dir, lvm or zfs)
+LXD\_BACKEND                    | dir                       | What backend to test against (btrfs, dir, lvm or zfs, or RANDOM to pick a random one)
```

Comment:
only reason was to distinguish it from actual values, but it's not a big deal, I can change that to lowercase